### ISSUE TITLE
Error: couldn't find any trees; start:end : argument of length 0

### ISSUE THREAD

**Issue by jbernst** [danlwarren/RWTY#172]
Hello, 
I am currently trying to assess and visually show the convergence of 5 separate starbeast2 runs using the beauti & beast GUI.
In R, I set my working directory to a directory containing the following (log files, trees files, and tree files [TreeAnnotator]:
Homalopsidae_Run1.tree
Homalopsidae_Run2.tree
Homalopsidae_Run3.tree
Homalopsidae_Run4.tree
Homalopsidae_Run5.tree
species_Run1.trees
species_Run2.trees
species_Run3.trees
species_Run4.trees
species_Run5.trees
species_Run1.log
species_Run2.log
species_Run3.log
species_Run4.log
species_Run5.log

If I try `load.trees(file = ".",  format = "*beast")` then I get the error "Error in start:end : argument of length 0."
This is also the case if I try "beast" (even though this is from starbeast). 

I also tried running `load.multi(path= PATH,format = "*beast")` and then I get:
Error in load.multi(path = PATH, format = "*beast") : 
  Couldn't find any tree files.

The closest I get to running anything is if I switch the load.multi format to "beast" in which I get the following output:
Error in scan(file = file, what = what, sep = sep, quote = quote, dec = dec,  : 
  line 1366 did not have 6 elements.

I assume this has to do with the fact that I show "beast" instead of "*beast", but unfortunately I haven't had success with either. Are there any ideas of what I may be doing wrong? I've tried a few derivations of the above code using different parameters, but nothing has worked. It would be awesome to show that all of my runs are similar (in Tracer, all parameter distributions are near identical, but I would love to show some cool-looking RWTY graphics).

Thank you!

Best,
Justin

**Comment by danlwarren** [comment]
I'm going to guess that it's something to do with the fact that there are "trees" files and "tree" files, and maybe rwty is looking for log files that go with the .tree files and failing?  Just as an experiment, can you remove the .tree files from the directory and give load.multi() another shot?

**Comment by jbernst** [comment]
Thanks for the reply! I went in and checked the .log files, and it seems one of them had some sort of formatting issue (very possible I accidentally types or tabbed something in the file and then saved it). I went back to the original file and used the load.multi code again (using "beast", "*beast" still didn't work) and it ran successfully. It says the `All trees must have the same tip labels` but I presume that somewhere I made a typo (I had to relabel the tips, so I must have made a mistake somewhere.

Thank you!
